docker/bitcoind/start-bitcoind.sh

```diff
@@ -3,6 +3,7 @@
 set -e
 
 MINER=${MINER:-false}
+MINER_LOCAL_REGTEST=${MINER_LOCAL_REGTEST:-false}
```

Comment:
@bradleystachurski we have a few options for this env variable.

`MINER_DEV=true`
or
`MINER_ENV=local`

Its probably worth it for us to avoid having REGTEST in the environment variable, just to make the API to these files more generic. What do you think?